README.md

```diff
@@ -1,2 +1,118 @@
-# lorawan-gateway
-viam module for a sx1302 lorawan gateway.
+# LoRaWAN Module
+
+Viam module for receiving data from LoRaWAN sensors.\
+`sx1302-gateway`: sensor model for a sx1302 lorawan concentrator hat connected to a raspberry pi.\
+`node`: sensor model for the end nodes sending data to the gateway.
+
+Compatible with:
+- US915 frequency band
+- Class A Devices
+- LoraWAN MAC version 1.0.3
+
+## Hardware Requirements
+
+- Raspberry Pi (any model with GPIO pins)
+- SX1302 Gateway HAT/concentrator board
+- US915 LoRaWAN sensors
+
+## Configuration
+
+### Gateway Configuration (viam:sensor:sx1302-gateway)
```

Comment:
Follow up -  this into two Configuration sections at the same level as the current Configuration section - one for each model (the gateway and the sensor) with configuration attributes, sample jsons and example jsons to copy for each.

Here's an example of a really cool readme - https://github.com/viam-modules/viam-camera-oak